js/tetris.js

```diff
@@ -175,6 +213,8 @@ function valid( offsetX, offsetY, newCurrent ) {
 }
 
 function newGame() {
+    score = 0;
+    document.getElementById("score").innerHTML = "Score: " + score.toString();
```

Comment:
Not sure how to move this to renderer without creating a reference, since renderer doesn´t get information about a new Game or ?